serverless/aws/protomaps-template.yaml

```diff
@@ -0,0 +1,187 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: CloudFormation template to create a protomaps infraestructure to serve tiles.
+Parameters:
+  BucketName:
+    Description: 'The name of the S3 bucket where you will store pmtiles files to be served (must be globally unique)'
+    Type: String
+
+  CodeBucketName:
+    Description: 'The S3 bucket name where the Lambda function code is stored (e.g., lambda-protomaps-code)'
+    Type: String
+
+  CodeKey:
+    Description: 'The S3 key for the Lambda function code (e.g., lambda_function.zip)'
```

Comment:
I think we can hardcode this as `lambda_function.zip` and require that to be the key name, instead of needing user configuration

Reply:
Added a similar comment below, but an alternative to the ZipFile would be to internally manage this:
```
# Example of inline code
Code:
        ZipFile: |
          const { S3Client, ListBucketsCommand } = require("@aws-sdk/client-s3");
          const s3 = new S3Client({ region: "us-east-1" }); // replace "us-east-1" with your AWS region

          exports.handler = async function(event) {
            const command = new ListBucketsCommand({});
            const response = await s3.send(command);
            return response.Buckets;
          };
```

This would be challenging if there are any need for lambda layers with dependencies which cannot be easily minified in JS.